packages/legacy/core/App/screens/PINEnter.tsx

```diff
@@ -70,6 +70,12 @@ const PINEnter: React.FC<PINEnterProps> = ({ setAuthenticated, usage = PINEntryU
       ...TextTheme.popupModalText,
       marginVertical: 5,
     },
+    image: {
+      ...PINEnterTheme.image,
```

Comment:
@thiagoromanos  PINEnterTheme.image created

Reply:
Nice work. That can make the whole PINEnter screen more theme flexible.